forch/forchestrator.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, config):
             _CONFIG_HASH_VERIFICATION_TIMEOUT_SEC_DEFAULT)
 
         self._lock = threading.Lock()
+        self._timer_lock = threading.Lock()
```

Comment:
NIT: Should the name be more specific? Like config_change_timer_lock or something?